catalog-rest-service/src/main/resources/json/schema/metadataIngestion/workflow.json

```diff
@@ -5,6 +5,32 @@
   "description": "OpenMetadata Ingestion Framework definition.",
   "type": "object",
   "definitions": {
+    "azureSSOConfig": {
+      "description": "Azure SSO client security configs.",
+      "type": "object",
+      "properties": {
+        "secretKey": {
```

Comment:
can we use the name as `clientSecret` here ? This is the standard term used by Azure!

Reply:
Yes, will modify it likewise